ci/ray_ci/test_container.py

```diff
@@ -0,0 +1,56 @@
+import sys
+
+import pytest
+from unittest import mock
+from typing import List
+
+from ci.ray_ci.container import run_command, run_tests
+
+
+class MockPopen:
```

Comment:
not a fan of these mocks, but maybe another time (or when you are in another country..)